r-cran/src/DESCRIPTION

```diff
@@ -0,0 +1,36 @@
+Package: rsparkling
+Title: R Interface for H2O Sparkling Water
+Version: 0.0.0
+Authors@R: c(
+  person("Jakub", "Hava", role = c("aut", "cre"), email = "jakub@h2o.ai"),
+  person("Navdeep", "Gill", role = c("aut"), email = "navdeep@h2o.ai"),
+  person("Erin", "LeDell", role = c("aut"), email = "erin@h2o.ai"),
+  person("Michal", "Malohlava", role = c("aut"), email = "michal@h2o.ai"),
+  person("JJ", "Allaire", role = c("aut"), email = "jj@rstudio.com"),
+  person("H2O.ai", role = c("cph")),
+  person("RStudio", role = c("cph")))
+Author: Jakub Hava [aut, cre],
+    Navdeep Gill [aut],
+    Erin LeDell [aut],
+    Michal Malohlava [aut],
+    JJ Allaire [aut],
+    H2O.ai [cph],
+    RStudio [cph]
+Maintainer: Jakub Hava <jakub@h2o.ai>
+Imports:
+    utils,
+    sparklyr (>= 0.3),
```

Comment:
Should we be more restrictive about versions of dependant packages? sparklyr 0.3 is 4+ years old.

Reply:
I just copied this from the main RSparkling package -> but agree, we should specify newer version here and in the main rsparkling package

Reply:
I was thinking that it might be actually better to do these in another JIRA, WDYT?